README.md

```diff
@@ -153,8 +153,8 @@ If, for some reason, the command returns a message asking for a minimum terminal
          Options
 
          -h, --help      Displays this information message
-         -d, --data      Selects a non-default data path
-	 -v, --version   Outputs the program version
+         -d, --data      Selects a non-default data path to game's assets and "share" folder
+	 	 -v, --version   Outputs the program version
```

Comment:
I would suggest to keep `-v, --version` aligned with the other options on README. What do you think?

Reply:
You're talking about the spacing (number of tabs/spaces)? 
For me it was aligned , but I can double check and fix it if that's what you're talking about!

Reply:
I believe it's fixed now! Can you confirm @math-araujo ?

Reply:
Yeah, it's fixed now! Thanks :)

Reply:
Nice :)
I can't review the PR so if you want to merge this request for us, it'd be great